common/helpers/g2p.py

```diff
@@ -0,0 +1,18 @@
+
+def get_g2p_parameters():
```

Comment:
It seems I misunderstood your idea of having a g2p helper. I thought that you want to do a generic "pipeline" here, that can also be used for switchboard, and then call this pipeline for each librispeech or switchboard corpus with parameters that are passed from the outside.

Because why would you have the default args here again? You could remove this and get exactly the same pipeline.

Reply:
Yes that would be good, because otherwise point (2) would be difficult. Do you think it makes sense to have a helper for training as well (so the conversion and train job)?

Reply:
> Do you think it makes sense to have a helper for training

Definitely, yes.

Reply:
+1 from my side for a generic g2p pipeline helper. 
The helper pipeline then would then get the g2p_args dict (so the user can easily configure settings) together with a train lexicon, a seed lexicon and a OOV list to add to the train lex.

Then in the librispeech part you would call something like
```
g2p_args = get_g2p_parameters()
train_lex = get_bliss_lexicon()
seed_lex = train_lex

for corpus_name, bliss_corpus in bliss_corpus_dict.items():
    if "train" in corpus_name:
        oov_file = ExtractOovWordsFromCorpusJob(bliss_corpus, seed_lex)
        seed_lex = my_g2p_pipeline(
            train_lex=train_lex,
            seed_lex=seed_lex,
            oov_file=oov_file,
            **g2p_args)
        augmented_bliss_corpora[corpus_name] = seed_lex
````
i.e. always the same lexicon is used to train the g2p model s.t. Sisyphus will reuse the Job.
and the `seed_lex` is successively extended by the oovs of the respective next larger corpus.

Reply:
I saw your comment too late. I already did some changes. Now we have a g2p_augmenter class that can be used for any lexicon. The class gets either a pre-trained model or in case no path is provided will train one model and then apply and give back the augmented lexicon when the relative is called.

Reply:
I see, this is already very close to my suggestion. 
I am only missing the option to have different lexica for training the g2p model and as seed for applying it.
(e.g. in cases when you continuously get more data and want to add new OOV at the end of the lexicon)

Reply:
Done -> [here](https://github.com/rwth-i6/i6_experiments/pull/15/commits/57b9dfdcffa5caa6a23cb918c5eb5ccfdbe4edb5)